### ISSUE TITLE
Suggestion: Automatically stop torrents.

### ISSUE THREAD

**Issue by Crayder** [kzahel/jstorrent-legacy-app#158]
There should be a few ways to automatically stop torrents, like in uTorrent.

**Comment by CaptainK77450** [comment]
clik the STOP button

**Comment by Crayder** [comment]
Yeah, one at a time when each finishes? That's annoying. Especially when you use torrents as much as I do.

**Comment by kzahel** [comment]
The legacy Chrome App is no longer maintained. I've rebuilt JSTorrent from scratch — the new version has auto-stop built in. Get it at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
